Makefile

```diff
@@ -48,7 +48,7 @@ ARCH ?= $(shell go env GOARCH)
 GOARCH = ${ARCH}
 DUMB_ARCH = ${ARCH}
 
-GOBUILD_FLAGS ?= -v
+GOBUILD_FLAGS :=
```

Comment:
Tested with both `make build` and `make build GOBUILD_FLAGS='-v -a'`.
Does what's expected in both scenarios using a pretty old `make` version (3.81, mac OS).